ni_measurementlink_service/_internal/discovery_client.py

```diff
@@ -144,30 +183,41 @@ def register_measurement_service(
         except Exception:
             _logger.exception("Error in registering with discovery service.")
             raise
-        return True
 
-    def unregister_service(self) -> bool:
-        """Un-registers the measurement service from the discovery service.
+    def unregister_service(self, registration_id: str = "") -> bool:
+        """Unregisters the specified service from the discovery service.
 
-        Should be called before the service is closed.
+        This method should be called before the service exits.
+
+        Args:
+        ----
+            registration_id: The registration ID returned from register_service.
+                This argument should be omitted after calling the deprecated
+                register_measurement_service method.
 
         Returns
         -------
```

Comment:
This formatting is wrong. Google docstrings use "Args:", "Returns:", etc. with a colon. The underlining is for rST titles which are not needed in Google docstrings.